test/components/drawer/drawer.visual-diff.js

```diff
@@ -0,0 +1,28 @@
+import puppeteer from 'puppeteer';
+import { VisualDiff } from '@brightspace-ui/visual-diff';
+
+describe('d2l-labs-drawer', () => {
+
+	const visualDiff = new VisualDiff('drawer', import.meta.url);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser);
+		await page.goto(`${visualDiff.getBaseUrl()}/test/drawer.visual-diff.html`, { waitUntil: ['networkidle0', 'load'] });
+		await page.bringToFront();
+	});
+
+	beforeEach(async() => {
+		await visualDiff.resetFocus(page);
+	});
+
+	after(async() => await browser.close());
+
+	it('passes visual-diff comparison', async function() {
```

Comment:
Would definitely be good to eventually have diffs for the drawer in different configurations, states. 😄